thirdparty/clamav-db/Makefile

```diff
@@ -11,9 +12,9 @@ mcvd := main.cvd
 bcvd_file := $(SRC_DIR)/$(bcvd)
 dcvd_file := $(SRC_DIR)/$(dcvd)
 mcvd_file := $(SRC_DIR)/$(mcvd)
-burl := http://database.clamav.net/$(bcvd)
-durl := http://database.clamav.net/$(dcvd)
-murl := http://database.clamav.net/$(mcvd)
+burl := https://files.zimbra.com/downloads/clamav-database/$(pvers)/$(bcvd)
```

Comment:
don't we have a public url from clamav for this?

Reply:
we have but clamav public url not allowed to directly download using wget and curl.
https://github.com/Cisco-Talos/clamav/issues/585#issuecomment-1127863490